src/components/DownloadFileButton.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <a :href="`/api/uploads/${upload.id}/download`" class="btn btn-primary ml-2">
```

Comment:
You could try adding the class `btn-sm` as well.  Maybe that'll help prevent content from wrapping quite so easily?